docs/opentelemetry/otel-fields-mapping.asciidoc

```diff
@@ -0,0 +1,2405 @@
+The following table gives an overview of mappings between individual ECS fields (in ECS version `9.0.0-dev`)
```

Comment:
do you think it makes sense to have all the fields from ecs here to make table more comprehensive that one can see how many fields are yet to add to semconv?

Reply:
That's why I added that other summary page with the table showing numbers per namespace

Reply:
yeah, I think I wanted to have it in one place but we could say that every particular namespace will have it and I can see exact fields there. So I’m ok with current view as well